src/index.html

```diff
@@ -11,7 +11,188 @@
+            type="date" 
+            name="birthday"
+            id="birthday"
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            name="terms"
+            id="terms"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="group-of-fields">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input 
+            type="email" 
+            name="email" 
+            id="email"
+            placeholder="email@example.com" 
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input 
+            type="password" 
+            name="password"
+            id="password"
+            minlength="5" 
+            maxlength="15" 
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="group-of-fields">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="form-field">
+          Do you love cats?
+          <label>
+            <input 
+              type="radio" 
+              name="love_cats"
+              id="loveCatsY" 
+              value="Yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input 
+              type="radio" 
+              name="love_cats" 
+              id="loveCatsN"
+              value="No"
+            >
+            No
+          </label>
+        </label>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input 
+            type="color" 
+            name="favouriteColor"
+            id="favoriteColor"
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input 
+            type="time" 
+            name="bedtime"
+            id="bedtime"
+            step="2"
+          >
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select 
+            name="favouriteCar" 
+            id="favoriteCar"
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field">
+          How do you rate our work?
+          <input 
+            type="range" 
+            name="mark"
+            id="mark" 
+            min="1" 
+            max="10" 
+            value="1"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="group-of-fields">
+        <legend>Additional info:</legend>
+
+        <label class="form-field">
+          Comments:
+          <textarea 
+            name="comment" 
+            id="comment"
+            maxlength="100"
+          >
+          </textarea>
```

Comment:
```suggestion
          ></textarea>
```
to prevent this
<img width="321" alt="image" src="https://user-images.githubusercontent.com/53611910/201548382-1fa3b6f1-5f85-4536-8ccb-b0db70d43aea.png">